deepgram/clients/live/v1_client.py

```diff
@@ -40,66 +43,148 @@ def __init__(self, base_url, api_key, headers):
       for handler in self._event_handlers[event]:
           handler(*args, **kwargs)
 
-  async def _start(self) -> None:
-      async for message in self._socket:
+  def _listening(self) -> None:
+      while True:
         try:
+            self.lock_exit.acquire()
+            myExit = self.exit
+            self.lock_exit.release()
+            if myExit:
+                return
+            
+            message = self._socket.recv()
+            if len(message) == 0:
+            #    print("empty message")
+               continue
+            
             data = json.loads(message)
             response_type = data.get("type")
-            if response_type == LiveTranscriptionEvents.Transcript.value:
-                await self._emit(LiveTranscriptionEvents.Transcript, data)
-            if "metadata" in data:
-                await self._emit(LiveTranscriptionEvents.Metadata, data["metadata"])
-        except json.JSONDecodeError as e:
-            await self._emit(LiveTranscriptionEvents.Error, e.code)
+            # print(f"response_type: {response_type}")
+
+            match response_type:
+                case LiveTranscriptionEvents.Transcript.value:
+                    result = LiveResultResponse.from_json(message)
+                    self._emit(LiveTranscriptionEvents.Transcript, result=result)
+                case LiveTranscriptionEvents.Metadata.value:
+                    result = MetadataResponse.from_json(message)
+                    self._emit(LiveTranscriptionEvents.Metadata, metadata=result)
+                case LiveTranscriptionEvents.Error.value:
+                    result = ErrorResponse.from_json(message)
+                    self._emit(LiveTranscriptionEvents.Error, error=result)
+                case _:
+                    error: ErrorResponse = {
+                        'type': 'UnhandledMessage',
+                        'description': 'Unknown message type',
+                        'message':  f'Unhandle message type: {response_type}',
+                        'variant':  '',
+                    }
+                    self._emit(LiveTranscriptionEvents.Error, error)
+
+        except Exception as e:
+            # print(f"Exception in _listening: {e}")
+            if e.code == 1000:
+                # print("Websocket closed")
+                return
+            
+            error: ErrorResponse = {
+                'type': 'Exception',
+                'description': 'Unknown error _listening',
+                'message':  f'{e}',
+                'variant':  '',
+            }
+            self._emit(LiveTranscriptionEvents.Error, error)
+
+  def _processing(self) -> None:
+    while True:
+        try:
+            time.sleep(PING_INTERVAL)
+
+            self.lock_exit.acquire()
+            myExit = self.exit
+            self.lock_exit.release()
+            if myExit:
+                return
+
+            # deepgram keepalive
```

Comment:
I didn't think we would be introducing the default KeepAlive until we had discussed whether it breaks billing on lag. Did we get an answer?

Reply:
I thought this was to be included. If not, I can no longer make it the default and put it behind an option flag.

Reply:
Since this is early on (still dont have an alpha yet), I will follow up with that on a subsequent PR. I want to get an alpha out the door ASAP.

Reply:
The PR was updated to disable by default but optional be enabled